index.php

```diff
@@ -56,16 +106,76 @@ function gutenberg_register_scripts() {
  * @since 0.1.0
  */
 function gutenberg_scripts_and_styles( $hook ) {
-	if ( 'toplevel_page_gutenberg' === $hook ) {
-		// Scripts
-		wp_register_script( 'gutenberg-content', plugins_url( 'post-content.js', __FILE__ ) );
-		wp_enqueue_script( 'wp-editor', plugins_url( 'editor/build/index.js', __FILE__ ), array( 'wp-blocks', 'wp-element', 'gutenberg-content' ), false, true );
-		wp_add_inline_script( 'wp-editor', 'wp.editor.createEditorInstance( \'editor\', { content: window.content } );' );
-
-		// Styles
-		wp_enqueue_style( 'wp-editor-font', 'https://fonts.googleapis.com/css?family=Noto+Serif:400,400i,700,700i', false );
-		wp_enqueue_style( 'wp-editor', plugins_url( 'editor/build/style.css', __FILE__ ) );
+	if ( 'toplevel_page_gutenberg' !== $hook ) {
+		return;
 	}
+
+	/**
+	 * Scripts
+	 */
+
+	// The editor code itself
+	wp_enqueue_script(
+		'wp-editor',
+		plugins_url( 'editor/build/index.js', __FILE__ ),
+		array( 'wp-blocks', 'wp-element', 'gutenberg-content' ),
+		false, // $ver
+		true   // $in_footer
+	);
```

Comment:
Here and elsewhere, I've restructured the code in this function to avoid long lines, which are really detrimental to code readability.  I'd recommend that we do this throughout the project, and add rule(s) to our linting setup(s) accordingly.